content/rs/concepts/memory-architecture/redis-flash.md

```diff
@@ -128,10 +128,8 @@ than extending RAM in a number of ways.
 To create Redis on Flash databases you must meet the following
 prerequisites:
 
-- Have ephemeral and persistent flash memory mounted on Redis
-    Enterprise Software nodes with the proper disk size, per the [Redis
-    on Flash memory
-    guidelines]({{< relref "/rs/administering/designing-production/hardware-requirements#production-environment-hardware-requirements" >}}).
+- Have [ephemeral and persistent storage]({{<relref "/rs/administering/designing-production/persistent-ephemeral-storage.md">}}) mounted on Redis
+    Enterprise Software nodes with the proper disk size.
```

Comment:
This is good. Any interesting in fixing the "Have all..." construction? To me this is awkward.

Reply:
I can change it to simple statements like: 
"Ephemeral and persistent storage is mounted on Redis Enterprise Software nodes with the proper disk size. "

Reply:
I think that would be better.

Reply:
@kaitlynmichaelredislabs I recommend fixing the format of the page so that each line contains a logical segment of a sentence. For example:

> To create Redis on Flash databases you must meet the following
> prerequisites:

Change to:

> To create Redis on Flash databases you must meet the following prerequisites: